### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jovany-wang**
https://github.com/ray-project/rayfed/issues/51
https://github.com/ray-project/rayfed/issues/59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let your write your federated and distributed computing applications like writing a single machine program.
+
+- **Multiple Controller Execution Mode**  
+  
+  The RayFed job can be ran in single controller mode when developing and debugging, while it will be ran in multiple controller mode in production without any code change.
```
Comment (NKcqx) [inline README.md:12]: "be ran" -> "be run"

File: README.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
 # RayFed
-A multiple parties joint, distributed execution engine on the top of Ray.
+A multiple parties joint, distributed execution engine based on Ray, to help build your own federated learning frameworks in minutes.
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let your write your federated and distributed computing applications like writing a single machine program.
```
Comment (NKcqx) [inline README.md:8]: Let you write your federated and distributed computing applications like a single-machine program.
* "Let your" -> "Let you"
* "like writing a " -> "like a"
* (optional) "single machine" -> "single-machine"

File: README.md (hunk 1 above)
Comment (jovany-wang) [reply README.md:12]: Good catch

File: README.md (hunk 2 above)
Comment (jovany-wang) [reply README.md:8]: Great.

**Comment by jovany-wang** [comment]
@NKcqx Comments were addressed.

File: README.md (hunk 1 above)
Comment (NKcqx) [reply README.md]: The RayFed job can be run in the single-controller mode for developing and debugging and the multiple-controller mode for production without code change

File: README.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+
+## Features
+- **Ray Native Programming Pattern**  
+  
+  Let you write your federated and distributed computing applications like a single-machine program.
+
+- **Multiple Controller Execution Mode**  
+  
+  The RayFed job can be run in single controller mode when developing and debugging, while it will be run in multiple controller mode in production without any code change.
```
Comment (NKcqx) [inline README.md]: My suggestion: "The RayFed job can be run in not only the single-controller mode for developing and debugging but also in the multiple-controller mode for production scenarios without any code change"

Besides, I think a short intro for the concept of "single-" & "multi-" controller is needed. I will follow up on this kinds of contents.

Comment (jovany-wang) [reply README.md]: Nice suggestion.

**Review by NKcqx — APPROVED** [review]

**Pull request merged**